src/os/filestore/os_xattr.h

```diff
@@ -24,6 +24,10 @@ extern "C" {
 # define ENOATTR ENODATA
 #endif
 
+#ifdef _WIN32
+#include <crtdefs.h>
```

Comment:
i assume that the libc used by MinGW also complies to at least C89/C90. so per https://en.cppreference.com/w/c/types/size_t, `stddef.h` should suffice.

Reply:
Yep, that's right. I'll use stddef.h instead.